### ISSUE TITLE
Disable spell check between $$ signs

### ISSUE THREAD

**Issue by tlnagy** [tpope/vim-markdown#70]
It would be nice to have an option to disable spell check within $$ signs because these commonly delimit math so something like 

$$ \frac{x}{y} $$

would have `frac` underlined as being misspelled, which is annoying. Similarly underscores are common in latex formulae and it would be nice if they were escaped as well.

**Comment by fikovnik** [comment]
Indeed, this would be great. Even cooler would be to have it syntax highlighted :-)
